content/06.methods.md

```diff
@@ -53,41 +52,41 @@ For serum-free media conditions, we plated cells at minimum density of 1×10<sup
+All RNA used for library prep had a minimum RIN of seven, but no QC thresholds were implemented for the DNA.
+For library preparation, 500 ng of nucleic acids were used as input for RNA-Seq, WXS, and targeted DNA panel (panel) sequencing.
 RNA library preparation was performed using the TruSeq RNA Sample Prep Kit (Illumina, #FC-122-1001) and the exome prep was performed using KAPA Library Preparation Kit (Roche, #KK8201) using Agilent's SureSelect Human All Exon V5 backbone with custom probes.
-The targeted DNA panel developed by Ashion (formerly known as the GEM Cancer panel) consisted of exonic probes against 541 cancer genes.
+The targeted DNA panel developed by Ashion Analytics (formerly known as the GEM Cancer panel) consisted of exonic probes against 541 cancer genes.
 Both panel and WXS assays contained 44,000 probes across evenly spaced genomic loci used for genome-wide copy number analysis.
 For the panel, additional probes tiled across intronic regions of 22 known tumor suppressor genes and 22 genes involved in common cancer translocations for structural analysis.
 All extractions and library preparations were performed according to manufacturer's instructions.
 
 ##### CBTN samples
 
 Blood, tissue, and cell line DNA/RNA extractions were performed at the Biorepository Core at CHOP.
-Briefly, 10-20 mg frozen tissue, 0.4-1ml of blood or 2×10^6 cells pellet was used for extractions.
+Briefly, 10-20 mg frozen tissue, 0.4-1ml of blood, or 2e6 **TODO just a note I did this for consistency since main text uses `e`** cells pellet was used for extractions.
 Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz settings using 5 mm steel beads (cat# 69989, Qiagen).
 Both tissue and cell pellets processes included a CHCl3 extraction and were run on the QIACube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen).
 Blood was thawed and treated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).
 DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).
-The NantHealth Sequencing Center, BGI at CHOP, or the Genomic Clinical Core at Sidra Medical and Research Center, performed library preparation and sequencing.
-Briefly, DNA sequencing libraries were prepared for tumor and matched-normal DNA using the KAPA HyperPrep kit (cat# 08098107702, Roche); tumor RNA-Seq libraries were prepared using KAPA Stranded RNA-Seq with RiboErase kit (cat# 07962304001, Roche).
+The NantHealth Sequencing Center, BGI at CHOP **TODO: Should this say BGI@CHOP?**, or the Genomic Clinical Core at Sidra Medical and Research Center performed library preparation and sequencing.
```

Comment:
> It could - BGI at CHOP doesn't exist anymore, so we can just keep with what's in the histologies file to be consistent. Not sure what it ever officially was...

Ah!! It's unclear official status is not super surprising...